assignment3.txt

```diff
@@ -0,0 +1,26 @@
+Emma Ladouceur
+
+Assignment Answers
+
+
+1. I dont' think it is valid.
+	First after the %d what is the code printing? That means it should be preparing for an integer and there is nothing defined nor is there a variable name following in. 
+	Secondly, there is a semicolon before the i.
+
+
+2. 
+There should not be a semi colon after the first set of () because it will go straight to the {}.
+
+
+3. I'm still a little confused about this, but I'm going to try it out.
+
+i. There are no factorials or squares or cubes or logs so I would say this is just 0(1) in terms of complexity.
```

Comment:
for loops that run from 0 to n are always of the time complexity O(n) - and when they're nested together, you multiply their time complexities. So in this example, the time complexity is O(n^2) because n * n = n^2